tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -62,13 +65,42 @@ def get_plugin_apps(self):
 
   def is_active(self):
     """The scalars plugin is active iff any run has at least one scalar tag."""
+    if self._db_connection_provider:
+      # The plugin is active if one relevant tag can be found in the database.
+      db = self._db_connection_provider()
+      cursor = db.execute(
+          'SELECT 1 FROM Tags WHERE Tags.plugin_name = ? LIMIT 1',
```

Comment:
[Optional] Any particular you chose not to write `SELECT COUNT(*) FROM Tags WHERE Tags.plugin_name = ?`? Did `EXPLAIN` say this was better?

Reply:
The reason I chose this query instead of an alternative one using `COUNT(*)` is that I think the latter would require a linear scan of rows. This current query halts as soon as one relevant one is discovered.